blog/2022-08-31-consensus.md

```diff
@@ -1,3 +1,15 @@
+---
+title: Consensus Team Update
+slug: 2022-08-31-consensus
+authors:
+  - name: Damian Nadales
```

Comment:
Do we need to set this once in one post and it shows in all the posts? I wonder if we could set this somewhere else. Maybe we need to document this feature in the README as well.

Reply:
Good point, there's the `authors.yml` file that allows only given a nickname.